lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -8,7 +8,8 @@ class DatabaseAdmin < HighLine
       LOCAL_FILE     = "Local file".freeze
       NFS_FILE       = "Network File System (NFS)".freeze
       SMB_FILE       = "Samba (SMB)".freeze
-      FILE_OPTIONS   = [LOCAL_FILE, NFS_FILE, SMB_FILE, CANCEL].freeze
+      S3_FILE        = "Amazon S3 (S3)".freeze
```

Comment:
Why is this a different string from the `RESTORE_S3` constant above? If we need the one in `bin/appliance_console` then these should definitely be the same.

Reply:
This is the right one...